packages/shopify-api/future/flags.ts

```diff
@@ -21,3 +24,24 @@ export type FeatureEnabled<
     ? true
     : false
   : false;
+
+export function logDisabledFutureFlags(
```

Comment:
Related to your PR in https://github.com/Shopify/shopify-app-js/pull/679

I think it would be confusing to see these API logs when you are using shopify-app-remix. Because when you are using that you can't turn on these API flags themselves. So we would be logging these error without folks being able to action them.

Reply:
I am wondering if we could add some sort of flag, that we can turn off from the remix library.

Reply:
Added a new commit that should address this problem. Also noticed a small issue with the function so improved it slightly.